app/components/balance-donut/template.hbs

```diff
@@ -1,40 +1,42 @@
-<div class="donut-title">{{this.title}}</div>
+<div style={{this.style}} class="user-header-absence-balance balance-donut {{this.color}}">
```

Comment:
```suggestion
<div style={{this.style}} class="balance-donut {{this.color}}">
```

What is this `user-header-absence-balance` class for? Seems like a use case specific class which should not be in here.